modules/svg/data.js

```diff
@@ -121,6 +122,11 @@ export function svgData(projection, context, dispatch) {
         if (!gj) return null;
 
         if (gj.type === 'FeatureCollection') {
+            // Split geometry collection into seperate features
+            if (containsGeometryCollection(gj)) {
+                geometryCollectionToFeatures(gj);
+            }
```

Comment:
I think GeometryCollection could also appear as a single feature, outside of a FeatureCollection.  It should be possible to support that too by making `geometryCollectionToFeatures` more like `ensureFeatureID` in that it takes a single feature at a time and operates on it.